src/Compiler/pppars.fsy

```diff
@@ -3,6 +3,8 @@
 %{
 open FSharp.Compiler.DiagnosticsLogger
 
+#nowarn "3261"  // the generated code would need to properly annotate nulls, e.g. changing System.Object to `obj|null`
```

Comment:
What's the main obstacle here now?

Reply:
The workflow to get this into fslex/fsyacc first, which means getting nullness into LKG toolchain first.